src/helpers/create/workflows/create-templated-repo.js

```diff
@@ -5,11 +5,11 @@ import { createResourceFromWorkflow } from '~/src/helpers/create/workflows/creat
  * Creates a new GitHub repository from a template
  *
  * @param {{ db: import('mongodb').Db, logger: import('pino').Logger}} request
- * @param {string} workflow          - name of the workflow in cdpCreateWorkflows to use
- * @param {string} repositoryName    - name of the GitHub repo to be created
- * @param {{github: string}} team    - team that will own the repo
- * @param {string[]} githubTopics    - list of topics to add to the repo
- * @param {Object} extraInputs       - extra fields to pass to the job
+ * @param {string} workflow       - name of the workflow in cdpCreateWorkflows to use
+ * @param {string} repositoryName - name of the GitHub repo to be created
+ * @param {string} team           - team that will own the repo
```

Comment:
I wonder if it's worth calling this githubTeam now? We'll have to make sure we do `{team: githubTeam}` below